actix-api/src/main.rs

```diff
@@ -25,10 +25,10 @@ const OAUTH_TOKEN_URL: &str = std::env!("OAUTH_TOKEN_URL");
 const OAUTH_SECRET: &str = std::env!("OAUTH_CLIENT_SECRET");
 const OAUTH_REDIRECT_URL: &str = std::env!("OAUTH_REDIRECT_URL");
 const SCOPE: &str = "email%20profile%20openid";
-const AFTER_LOGIN_URL: &str = "http://localhost/";
 const ACTIX_PORT: &str = std::env!("ACTIX_PORT");
 const UI_PORT: &str = std::env!("TRUNK_SERVE_PORT");
 const UI_HOST: &str = std::env!("TRUNK_SERVE_HOST");
+const AFTER_LOGIN_URL: &'static str = concat!("http://localhost:", std::env!("TRUNK_SERVE_PORT"));
```

Comment:
I felt kinda iffy about this line, but it was the fastest way I found to append the TRUNK_SERVE_PORT to a string slice after [a brief bit of googling](https://stackoverflow.com/a/35158196). Let me know if there's a better way or if you prefer the accepted answer from that stack overflow post, I'm still very much a Rust newbie.

Reply:
imo this is fine.